lib/fizzy/execute.cpp

```diff
@@ -325,9 +325,6 @@ const FuncType& function_type(const Instance& instance, FuncIdx idx)
 bool invoke_function(const FuncType& func_type, uint32_t func_idx, Instance& instance,
     Stack<uint64_t>& stack, int depth)
 {
-    if (depth == CallStackLimit)
```

Comment:
I decided that it's better to move this check into `execute`, because the host calling exported functions via `ExternalFunction` now can call `execute` with arbitrary `depth` value.